src/Minigames/driller.js

```diff
@@ -289,31 +277,63 @@ Commands.driller = new Command("Dig deeper and deeper to find the treasures\n\n"
             break
         }
         case "repair": {
+            args[1] = (args[1] || "").toLowerCase()
+
+            const hpAmount = parseInt(args[1])
+
             const cost =
-                args[1] == 'max'
-                    ? 100 * EconomySystem.driller - DrillerGame.hp
-                    : parseInt(args[1]);
+                Math.ceil(
+                    args[1] == 'max'
+                        ? DrillerGame.max_health
```

Comment:
The reason i made it `maxHP - currentHP` is because `repair` doesnt subtract the cost if it's above `maxHP`. So, the cost of repairing with `max` is higher than doing it manually.

Example if you dont subtract it:
```
<maxHP: 100, currentHP: 50>

&driller repair max
<cost: 100, currentHP: 100>

&driller repair 50
<cost: 50, currentHP: 50>
```